src/model/Name.ts

```diff
@@ -0,0 +1,7 @@
+export default interface Name {
+  prefix?: string
+  givenName?: string
+  familyName?: string
+  suffix?: string
+  fullName?: string
```

Comment:
I am still not convinced having fullName as field: it could be very very dangerous into the wild.